packages/gestalt/src/SlimBanner.js

```diff
@@ -0,0 +1,100 @@
+// @flow strict
+import { type Node } from 'react';
+import Box from './Box.js';
+import Icon from './Icon.js';
+import Link from './Link.js';
+import Text from './Text.js';
+import Flex from './Flex.js';
+import { MESSAGING_TYPE_ATTRIBUTES } from './messaging.js';
+
+type Props = {|
+  /**
+   * Helper [Link](https://gestalt.pinterest.systems/link) to be placed after the message. See the [helperLink variant](https://gestalt.pinterest.systems/slimbanner#helperLink) to learn more.
+   */
+  helperLink?: {|
+    text: string,
+    accessibilityLabel: string,
+    href: string,
+    onClick: ({|
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+  |},
+
+  /**
+   * Label to describe the status icon’s purpose. See the [Accessibility guidelines](https://gestalt.pinterest.systems/slimbanner#Accessibility) for details on proper usage.
+   */
+  iconAccessibilityLabel?: string,
+  /**
+   * Main content of SlimBanner. Content should be [localized](https://gestalt.pinterest.systems/slimbanner#Localization).
+   *
+   */
+  message: string,
+  /**
+   * The type of status to display. See the [status variant](https://gestalt.pinterest.systems/slimbanner#Variants) to learn more.
+   */
+  status?: 'neutral' | 'error' | 'info' | 'warning' | 'success',
+  /**
+   * The type of SlimBanner. See the [type variant](https://gestalt.pinterest.systems/slimbanner#Variants) to learn more.
+   */
+  type?: 'default' | 'light',
```

Comment:
Maybe 'default' and 'bare', or 'lite'? I don't think "light" is the right word for this

Reply:
refactored to combined props

Reply:
I like "bare". I had also called it "borderless" the way Apple does for buttons without backgrounds, but that's confusing since technically, all of them don't have borders.